pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+Changes in the content of a [web page][] occur when at least one of the following occurs:
+
+- **visible changes:** Visible changes in content occur when the rendered pixels change in any part of the document that is currently within the viewport or that can be brought into the viewport via scrolling; or
+- **accessibility tree changes:** Changes in the content of the accessibility tree occur when any state, property or event of a node representing an [accessible object][] of the tree changes, or when any node is inserted in, or removed from the tree; or
+- **audible changes:** Audible changes in content occur when the audio rendered by the [web page][] change.
```

Comment:
"Viewport" and "accessibility tree" may need links to specs.

Reply:
We haven't been doing this in other rules, but why not :-)